packages/core/admin/admin/src/translations/fi.json

```
@@ -0,0 +1,594 @@
+  "Auth.form.button.login.strapi": "Kirjaudu sisään Strapilla",
+  "Auth.form.button.password-recovery": "Salasanan palautus",
+  "Auth.form.button.register": "Valmis aloittamaan",
+  "Auth.form.confirmPassword.label": "Salasanan vahvistus",
+  "Auth.form.currentPassword.label": "Nykyinen salasana",
+  "Auth.form.email.label": "Sähköposti",
+  "Auth.form.email.placeholder": "kai@doe.com",
+  "Auth.form.error.blocked": "Järjestelmänvalvoja on estänyt tilisi.",
+  "Auth.form.error.code.provide": "Koodi on virheellinen.",
+  "Auth.form.error.confirmed": "Tilisi sähköpostiosoitetta ei ole vahvistettu.",
+  "Auth.form.error.email.invalid": "Tämä sähköposti on virheellinen.",
+  "Auth.form.error.email.provide": "Käyttäjänimesi puuttuu.",
+  "Auth.form.error.email.taken": "Tämä sähköposti on jo käytössä",
+  "Auth.form.error.invalid": "Käyttäjänimesi tai salasanasi on virheellinen.",
+  "Auth.form.error.params.provide": "Tiedot ovat virheellisiä.",
+  "Auth.form.error.password.format": "Salasanasi ei voi sisältää symbolia `$` kolmea kertaa.",
+  "Auth.form.error.password.local": "Tällä tilillä ei ole salasanaa.",
+  "Auth.form.error.password.matching": "Salasanat eivät ole identtisiä.",
+  "Auth.form.error.password.provide": "Salasanasi puuttuu.",
+  "Auth.form.error.ratelimit": "Liian monta yritystä, yritä uudelleen minuutin kuluttua.",
+  "Auth.form.error.user.not-exist": "Tätä sähköpostiosoitetta ei ole olemassa.",
+  "Auth.form.error.username.taken": "Tämä nimi on jo käytössä",
+  "Auth.form.firstname.label": "Etunimi",
+  "Auth.form.firstname.placeholder": "Kai",
+  "Auth.form.forgot-password.email.label": "Anna sähköpostiosoitteesi",
+  "Auth.form.forgot-password.email.label.success": "Sähköpostin lähetys onnistui seuraavaan osoitteeseen",
+  "Auth.form.lastname.label": "Sukunimi",
+  "Auth.form.lastname.placeholder": "Doe",
+  "Auth.form.password.hide-password": "Piilota salasana",
+  "Auth.form.password.hint": "Salasanan tulee sisältää vähintään 8 merkkiä, 1 iso kirjain, 1 pieni kirjain ja 1 numero.",
+  "Auth.form.password.show-password": "Näytä salasana",
+  "Auth.form.register.news.label": "Pidä minut ajan tasalla tulevista uusista ominaisuuksista ja parannuksista (tekemällä näin hyväksyt {terms} ja {policy}).",
+  "Auth.form.register.subtitle": "Kirjautumistietojasi käytetään vain todentamiseen hallintaliittymässä. Kaikki tallennetut tiedot tallennetaan omaan tietokantaan.",
+  "Auth.form.rememberMe.label": "Muista minut",
+  "Auth.form.username.label": "Käyttäjänimi",
+  "Auth.form.username.placeholder": "Kai Doe",
+  "Auth.form.welcome.subtitle": "Kirjaudu sisään Strapi-tilillesi",
+  "Auth.form.welcome.title": "Tervetuloa!",
+  "Auth.link.forgot-password": "Unohditko salasanasi?",
+  "Auth.link.ready": "Valmis muodostamaan yhteyden?",
+  "Auth.link.signin": "Yhteys",
+  "Auth.link.signin.account": "Onko sinulla jo tili?",
+  "Auth.login.sso.divider": "Tai kirjaudu sisään",
```

Maybe a bit ambiguous as is? 🤷‍♂️ 

```suggestion
  "Auth.login.sso.divider": "Tai kirjaudu sisään käyttämällä ulkoista palvelua",
```